api/src/test/java/org/pmiops/workbench/utils/TimeAssertions.java

```diff
@@ -14,29 +16,51 @@
 
   public static final Duration DEFAULT_TOLERANCE = Duration.ofMillis(100);
 
-  public static void assertTimeWithinTolerance(OffsetDateTime actual, OffsetDateTime expected) {
-    assertTimeWithinTolerance(actual.toInstant(), expected.toInstant(), DEFAULT_TOLERANCE);
+  public static void assertTimeApprox(OffsetDateTime actual, OffsetDateTime expected) {
+    assertTimeApprox(actual.toInstant(), expected.toInstant(), DEFAULT_TOLERANCE);
   }
 
-  public static void assertTimeWithinTolerance(
+  public static void assertTimeApprox(
       OffsetDateTime actual, OffsetDateTime expected, Duration tolerance) {
-    assertTimeWithinTolerance(actual.toInstant(), expected.toInstant(), tolerance);
+    assertTimeApprox(actual.toInstant(), expected.toInstant(), tolerance);
   }
 
-  public static void assertTimeWithinTolerance(Instant actual, Instant expected) {
-    assertTimeWithinTolerance(actual, expected, DEFAULT_TOLERANCE);
+  public static void assertTimeApprox(Timestamp actual, Timestamp expected) {
+    assertTimeApprox(actual.toInstant(), expected.toInstant());
   }
 
-  public static void assertTimeWithinTolerance(
-      Instant actual, Instant expected, Duration tolerance) {
-    assertTimeWithinTolerance(actual.toEpochMilli(), expected.toEpochMilli(), tolerance.toMillis());
+  public static void assertTimeApprox(Timestamp actual, Timestamp expected, Duration tolerance) {
+    assertTimeApprox(actual.toInstant(), expected.toInstant(), tolerance);
   }
 
-  public static void assertTimeWithinTolerance(long actualEpochMillis, long expectedEpochMillis) {
-    assertTimeWithinTolerance(actualEpochMillis, expectedEpochMillis, DEFAULT_TOLERANCE.toMillis());
+  public static void assertTimeApprox(OffsetDateTime actual, Timestamp expected) {
+    // TODO: move to TimeMappers class after merging that
+    assertTimeApprox(actual, OffsetDateTime.ofInstant(expected.toInstant(), ZoneOffset.UTC));
   }
 
-  public static void assertTimeWithinTolerance(
+  public static void assertTimeApprox(
```

Comment:
This one is a little special, but somewhere I had a whole bunch of assertions that needed this combination of arguments.